client/src/components/Gallery.jsx

```diff
@@ -28,23 +38,53 @@ class Gallery extends React.Component{
     this.getRequest()
   }
 
+  openModal(e){
+    e.preventDefault();
+    this.setState({
+      modal: true
+    })
+  }
+
+  closeModal(e){
+    e.preventDefault();
+    this.setState({
+      modal: false
+    })
+  }
```

Comment:
Not a big deal, but you may be able to combine `openModal` and `closeModal` by doing a toggle between true and false. I don't fully understand your implementation, but just something to think about.
Roger

Reply:
Oh yeah, I'll check it out. Thanks!